src/i18n/baseConfig.ts

```diff
@@ -1,9 +1,12 @@
-import i18n, { InitOptions } from 'i18next'
+// @ts-check
```

Comment:
Why is this necessary in a TS file?

Reply:
Good catch.  This was from me running around.